blocks/accessories-category/accessories-category.css

```diff
@@ -47,7 +47,7 @@ max-width: 100%;
 
 @media screen and (min-width: 1280px) {
   .accessories-category > ul {
-    grid-template-columns: repeat(5, 1fr);
```

Comment:
I had made it 5 in case they added more accessories :-)

Reply:
yeah, but i wanted it to look like the live site. If they add more, its an easy change to fix it.

Reply:
Though it only looks correct on NA/EN... so i'll revert this on a diff update 👍